Add pie chart to represent governance (#26)
NoahSaso/dao-up: docs/pages/evaluating-campaigns.mdx

# Evaluating campaign trustworthiness

DAO Up! does a lot to make sure that backer funds are safe. Though at
the end of the day, technology can't solve all trust issues. When
backing a campaign on DAO Up!, here are some tips for making sure it is
safe.

1. Look at the percentage of governance power being allocated to
   campaign backers. Make sure that backers will have sufficient voting

>    power to influence decisions the DAO makes, such as withdraw money.

Reviewer: ```suggestion
   power to influence decisions the DAO makes. For example, withdrawing money.
```
